FORMAT.md

```diff
@@ -215,12 +215,12 @@ anything other than domain names and text strings, MUST not be quoted.
   "$DATE" and "$GENERATE" (and "$TTL" before RFC2308) are considered valid
   domain names in other implementations (based on what is accepted for domain
   names, see earlier points). It seems "$" is better considered a reserved
-  character (possibly limiting it's special special status to the start of the
+  character (possibly limiting its special status to the start of the
```

Comment:
two changes:
* removing a duplicated word which didn't seem to add value
* `it's` = `it is` which isn't the right concept here, replacing it with `its` (possessive)